x/slashing/types/codec.go

```diff
@@ -0,0 +1,30 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+)
+
+// RegisterCodec registers concrete types on codec
+func RegisterCodec(cdc *codec.Codec) {
+	cdc.RegisterConcrete(MsgUnjail{}, "cosmos-sdk/MsgUnjail", nil)
+}
+
+type Codec struct {
```

Comment:
We don't actually need a concrete type here anymore. See https://github.com/cosmos/cosmos-sdk/blob/master/x/distribution/types/codec.go for an example. You can just copy and modify.